packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -149,6 +149,40 @@ export interface IDataStore extends IFluidRouter {
 	 * the data store's entryPoint.
 	 */
 	readonly entryPoint?: IFluidHandle<FluidObject>;
+
+	/**
+	 * Issue a request against the DataStore for its "root" object aka its entryPoint. @see {@link IDataStore.entryPoint}.
+	 *
+	 * NOTE: Requesting "/" is idiomatic to some known consumers of Fluid Framework;
+	 * the root URL "/" will not route anywhere in Fluid natively.
+	 * This accomodation is provided as a temporary measure to ease the transition from requests to entryPoint,
+	 * where the paradigm requesting "/" is used.  This paradigm requires additional work to support (using requestHandler)
+	 * and should not be adopted if not already used - rather, use entryPoint directly.
+	 *
+	 * Refer to Removing-IFluidRouter.md for details on migrating from the request pattern to using entryPoint.
+	 *
+	 * @param request - Only requesting \{ url: "/" \} is supported, requesting arbitrary URLs is deprecated.
+	 */
+	request(request: { url: "/"; headers?: undefined }): Promise<IResponse>;
```

Comment:
Slight concern about documenting this as issuing a request for the entryPoint. The request handler that ends up servicing a call to this method is not required to return the same thing as `.entryPoint`, right? Same thing for `IContainer`.

Also, what's the rationale for keeping `headers` here? The test file suggests providing any headers should be deprecated, and that would be covered by having deprecated the more general version of `request()`, no?

Tangent: TIL `undefined` is a valid type.

Reply:
I don't remember why but I think I had to add that undefined param to get it to work. I'll revisit it.